docs/reference/requirement-sets/excel-api-requirement-sets.md

```diff
@@ -39,6 +39,586 @@ For more information about versions, build numbers, and Office Online Server, se
 - [Where you can find the version and build number for an Office 365 client application](https://support.office.com/article/version-and-build-numbers-of-update-channel-releases-ae942449-1fca-4484-898b-a933ea23def7)
 - [Office Online Server overview](https://docs.microsoft.com/officeonlineserver/office-online-server-overview)
 
+## Excel JavaScript preview API set
```

Comment:
Since technically "Preview APIs" do not belong to any *requirement set*, suggest rephrasing this section heading -> "Excel JavaScript preview APIs"